cluster/addons/cluster-loadbalancing/glbc/default-controller.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: glbc-default-backend
```

Comment:
If we need exactly one of these, why not run it as another container in the GLBC pod - simpler...